group_hooks.go

```diff
@@ -41,6 +42,7 @@ type GroupHook struct {
 	WikiPageEvents           bool       `json:"wiki_page_events"`
 	DeploymentEvents         bool       `json:"deployment_events"`
 	ReleasesEvents           bool       `json:"releases_events"`
+	SubGroupEvents           bool       `json:"subgroup_events"`
```

Comment:
```suggestion
	SubgroupEvents           bool       `json:"subgroup_events"`
```

Looking at the pattern of other names I suggest to *camel case* when *snake cased* :D

Reply:
... and don't get me wrong - I have the same urge to camel case to `SubGroup` ... but would favor consistency in this case 💃

Reply:
Agree with both these statements 😏 So lets keep _camel casing_ whatever the _snake cased_ names are. Even through in this case (no pun intended) I would also tent to write `SubGroup...`

Reply:
I was originally going to use `SubgroupEvents`, but this struct https://github.com/xanzy/go-gitlab/blob/1a0a4f28753dbb04978845336d90d402ab4b36b5/event_webhook_types.go#L894 was `SubGroupEvent`, so I was trying to be consistent with that casing. Should I change both places, or just here?

Reply:
Nice find! Please update both places then :) Thanks!

Reply:
Ok, I cleaned up those that were related to events. However, there are a few other places that use `SubGroup*` camel casing. To keep this PR focused, I left them alone for now.